### PR TITLE
[#98][UI] As a user, I can response to slider survey type

### LINKED ISSUE #98: [UI] As a user, I can response to slider survey type
## Why
The application supports various types of survey items. After the user fills in all required survey items, the survey can be submitted and mark the survey as completed.

## Acceptance Criteria
- Display answer section for `slider` survey type

image

- Store the user's response for survey submission

### PR DESCRIPTION
**Pull request by hanam1ni**
- Closes #98 

## What happened 👀

Add slider answer type.
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/14077479/181298091-d8ba3626-74b4-4404-b696-0cdc07cc9d83.mov

### CODE REVIEW

File: src/components/Slider/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+      fireEvent.keyDown(slider, {
+        keyCode: ARROW_LEFT_KEYCODE,
+      });
+
+      expect(onChange).toHaveBeenNthCalledWith(1, 1);
+      expect(onChange).toHaveBeenNthCalledWith(2, 0);
+    });
+  });
+});
```
Comment (malparty) [inline src/components/Slider/index.test.tsx:50]: Nice workaround using arrow keys to test!!! 💡

**Review by malparty — APPROVED** [review]
No comments :P Looks good to me! 
👏  Kudos for the `memo` and easy-to-read Slider component (yeah, as I often tell when it is hard to understand, this component was surprisingly clear and explicit, so I'd thought I should also notice it :D)

File: src/components/Slider/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import Slider from '.';
+
+const ARROW_LEFT_KEYCODE = 37;
+const ARROW_RIGHT_KEYCODE = 39;
+
+describe('Slider', () => {
+  describe('given labels', () => {
+    test('renders slider with label for each step', () => {
```
Comment (hoangmirs) [inline src/components/Slider/index.test.tsx:10]: Why don't you use `it` instead?
```suggestion
    it('renders the slider with labels for each step', () => {
```

**Review by hoangmirs — APPROVED** [review]
LGTM with a minor question

File: src/components/Slider/index.test.tsx (hunk 2 above)
Comment (hanam1ni) [reply src/components/Slider/index.test.tsx]: My bad, I get used to using `test` from Elixir testing 🙈 Updated in [e774410](https://github.com/hanam1ni/react-surveyor/pull/100/commits/e774410d372f878135fe4812da5913aa27098586)

**Review by malparty — APPROVED** [review]

**Pull request merged**